src/com/company/MyListTask2.java

```diff
@@ -0,0 +1,34 @@
+package com.company;
+
+import java.util.*;
+
+public class MyListTask2<T extends Comparable> {
+
+    public List<T> myListTask2 = new ArrayList<T>();
+
+    public void add(T t) {
+        myListTask2.add(t);
+    }
+
+    public T getMaxValue() {
+        T maxValue = myListTask2.get(0);
+        for (int i = 0; i < myListTask2.size(); i++) {
+            T value = myListTask2.get(i);
+            if (maxValue.compareTo(value) < 1) {
```

Comment:
Wrong logic.
If you want to check is the 'value' greater than the 'maxValue' you have to use the expression `value.compareTo(maxValue) > 0`